pkg/uhttp/wrapper_test.go

```diff
@@ -385,7 +387,7 @@ func TestWrapper_NewRequest(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			client := NewBaseHttpClient(http.DefaultClient)
+			client, _ := NewBaseHttpClient(ctx, http.DefaultClient)
```

Comment:
The tests could require.NoError() when creating base clients.

Reply:
Fixed